tuxemon/core/item.py

```diff
@@ -145,6 +147,33 @@ def advance_round(self):
         """
         return
 
+    def validate(self, target):
+        """Ensures that the target meets all conditions that the
+        item has on it's use.
+
+        :param user: The monster or object that is using this item.
+        :param target: The monster or object that we are using this item on.
+
+        :type user: Varies
```

Comment:
s/Varies/Any to conform with the `typing` module

Reply:
please change Varies to Any to conform with the typing module.  https://docs.python.org/3/library/typing.html#typing.Any